test/snapshot_test.rb

```diff
@@ -0,0 +1,143 @@
+require_relative 'spec_helper'
+require 'yast'
+require 'dbus'
+require_relative '../src/lib/snapper/snapshot.rb'
+
+describe Yast2::Snapper::Snapshot, "with default strategy" do
```

Comment:
Yeah, i had fixed it, because get into confusion. Really this test are based on SnapshotDbus strategy, so should be in a nested context

describe Yast2::Snapper::Snapshot do
  context "with SnapshotDbus strategy" do